File: Robot/encoderRead.py
import Adafruit_BBIO.ADC as ADC #Similar to analog read
import time as T

# Voltage Multiplier:
Volt = 1.8

# Setting previous read:
prevA = 0
prevB = 0

ADC.setup() #may need to put argument

while True:

	#OUTA = ADC.read_raw("P9_37")
	#OUTB = ADC.read_raw("P9_39")

	# Another option:
	#OUTA = round((ADC.read("P9_36") * Volt), 3)
	#OUTB = round((ADC.read("P9_33") * Volt), 3)
	OUTA = ADC.read("P9_36")
	OUTB = ADC.read("P9_33")
	
	#OUTA = "%.3f" % OUTA
	#OUTB = "%.3f" % OUTB
	'''
	if OUTA != prevA:
		print('OUTA:')
		print(OUTA)

	if OUTB != prevB:
		print('OUTB:')
		print(OUTB)
	'''

	print 'OUTA: ' + str(OUTA)
	#print 'OUTB: ' + str(OUTB)

	T.sleep(0.5)

'''
print('Printing value of OUTA:')
print(OUTA)
print('Printing value of OUTB:')
print(OUTB)
'''

File: Robot/encoderRead.js
#!/usr/bin/env node

var b = require('bonescript');

// Declaring variables
var OUTA = 'P9_37'; //analog in
var OUTB = 'P9_39'; //analog in

//while(0<10){
	// Setting pinout
	b.analogRead(OUTA, printStatusA);
	b.analogRead(OUTB, printStatusB);

//while(1===1){
	function printStatusA(x){
		console.log('x.value = ' + x.value);
		console.log('x.err = ' + x.err);
	}

	function printStatusB(y){
		console.log('y.value = ' + y.value);
		console.log('y.err = ' + y.err);
	}
//}
